test/QueryStrategies/batch_qs_test.jl

```diff
@@ -0,0 +1,212 @@
+@testset "BatchQueryStrategies" begin
+    BQS = OneClassActiveLearning.QueryStrategies.BatchQueryStrategies
+
+    dummy_data, dummy_labels = load_data(TEST_DATA_FILE)
+    classifier = SVDD.RandomOCClassifier(dummy_data)
+
+    empty_params = Dict{Symbol, Any}()
+    sequential_strategy = Dict{Symbol, Any}(
+        :type => :RandomPQs,
+        :param => Dict{Symbol, Any}()
+    )
+    batch_size = 5
+
+    function test_batch_qs(qs_type, params::Dict{Symbol, Any}, classifier, data, labels, candidate_indices)
+        @test_throws Union{MethodError, ArgumentError} initialize_qs(qs_type, classifier, data, empty_params)
+        qs = initialize_qs(qs_type, classifier, data, Dict(params))
+        for candidates in candidate_indices
+            batch_indices = select_batch(qs, data, labels, candidates)
+            @test length(batch_indices) == min(length(candidates), batch_size)
+            @test issubset(batch_indices, Set(candidates))
+        end
+    end
+
+    @testset "batch_qs utils" begin
+        @test all(BQS.normalize_weights(1, 1, 1) .≈ (1/3, 1/3, 1/3))
```

Comment:
What happens if weights are negative?

Reply:
Good catch. We also have to support something like `normalize_weights(-1, 1, 1) .≈ (-1/3, 1/3, 1/3))`. I have added this as a test case and fixed the normalization.

I wonder if we actually need the normalization of the weights in the first place?

Reply:
I am removing the weight normalization.